sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-generation.service.ts

```diff
@@ -497,4 +508,38 @@ export class ECertGenerationService {
       }
     }
   }
+
+  /**
+   * Calculate tuition remittance effective amount.
+   * @param disbursements all disbursements that are part of one e-Cert.
+   */
+  private calculateTuitionRemittanceEffectiveAmount(
+    disbursements: ECertDisbursementSchedule[],
+  ) {
+    for (const disbursement of disbursements) {
+      const award: Award[] = disbursement.disbursementValues?.map(
+        (eachDisbursementValue: DisbursementValue) => ({
+          valueType: eachDisbursementValue.valueType,
+          disbursedAmountSubtracted:
+            eachDisbursementValue.disbursedAmountSubtracted,
+          valueAmount: eachDisbursementValue.valueAmount,
+          effectiveAmount: eachDisbursementValue.effectiveAmount,
+        }),
+      );
+      const offering =
+        disbursement.studentAssessment.application.currentAssessment.offering;
+      const offeringCosts: OfferingCosts = {
+        actualTuitionCosts: offering.actualTuitionCosts,
+        programRelatedCosts: offering.programRelatedCosts,
+      };
+      const calculationType = MaxTuitionRemittanceTypes.Effective;
```

Comment:
I do not benefit from declaring the enum type to be passed as a method parameter. Avoiding it would make the method simpler, IMO.